pipeline/terraform/stack/service_image_id_minter.tf

```diff
@@ -0,0 +1,79 @@
+module "image_id_minter_queue" {
+  source          = "git::github.com/wellcomecollection/terraform-aws-sqs//queue?ref=v1.1.2"
+  queue_name      = "${local.namespace_hyphen}_image_id_minter"
+  topic_arns      = [module.merger_images_topic.arn]
+  aws_region      = var.aws_region
+  alarm_topic_arn = var.dlq_alarm_arn
+}
+
+module "image_id_minter" {
+  source          = "../modules/service"
+  service_name    = "${local.namespace_hyphen}_image_id_minter"
+  container_image = local.id_minter_image
+
+  security_group_ids = [
+    aws_security_group.service_egress.id,
+    aws_security_group.interservice.id,
+    var.rds_ids_access_security_group_id,
+  ]
+
+  cluster_name = aws_ecs_cluster.cluster.name
+  cluster_arn  = aws_ecs_cluster.cluster.id
+
+  namespace_id = aws_service_discovery_private_dns_namespace.namespace.id
+
+  env_vars = {
+    metrics_namespace    = "${local.namespace_hyphen}_image_id_minter"
+    messages_bucket_name = aws_s3_bucket.messages.id
+
+    queue_url       = module.image_id_minter_queue.url
+    topic_arn       = module.image_id_minter_topic.arn
+    max_connections = 8
+
+    logstash_host = local.logstash_host
+  }
+
+  secret_env_vars = {
+    cluster_url = "catalogue/id_minter/rds_host"
+    db_port     = "catalogue/id_minter/rds_port"
+    db_username = "catalogue/id_minter/rds_user"
+    db_password = "catalogue/id_minter/rds_password"
+  }
+
+  // The maximum number of connections to RDS is 90.
+  // Each id minter task is configured to have 8 connections
+  // in the connection pool (see `max_connections` parameter above).
+  // To avoid exceeding the maximum nuber of connections to RDS,
+  // the maximum capacity for *EACH* id minter  (ie the total
+  // maximum capacity of all ID minters) should be no higher than 5
+  max_capacity = 5
```

Comment:
You've updated the number but not taken advantage of it! Possibly I wrote a less-than helpful comment.

If each ID minter maxes its connections, we'd be using `8 * id_minter = 8 * 5 < 45` count connections. You can play with these numbers if you want more performance/to take advantage of the extra connections.

Reply:
I disagree! There are 2 ID minter services now, so you are missing a factor of 2 - however I see that my `*EACH*` isn't clear that I'm talking about the services. Will clarify.

Reply:
I wonder if having these numbers be computed on the fly might help?

```hcl
locals {
  id_minter_count       = 2
  id_minter_concurrency = 8
  db_max_connections    = 90
}

...

max_capacity = floor(local.db_max_connection / local.id_minter_count / local.id_minter_concurrency)
```

Reply:
Nice idea, will do - what a shame that terraform can't do integer linear programming